aspnetcore/migration/22-to-30.md

```diff
@@ -878,6 +878,10 @@ services.AddSignalR(hubOptions =>
 
 In ASP.NET Core 2.2, you could set the `TransportMaxBufferSize` and that would effectively control the maximum message size. In ASP.NET Core 3.0, that option now only controls the maximum size before backpressure is observed.
 
+#### SignalR assemblies in shared framework
+
+ASP.NET Core SignalR assemblies are no longer published to NuGet as packages. Instead, the assemblies are included in the `Microsoft.AspNetCore.App` shared framework, which is installed with the .NET Core SDK and runtime installers. See [Remove obsolete package references](#remove-obsolete-package-references) in this document.
```

Comment:
And then mention what I said previously about `Microsoft.AspNetCore.App` included in newer versions. At this   point, most using this doc will be going to .NET 6 or later and you don't want to waste their time thinking about `Microsoft.AspNetCore.App`

Reply:
@Rick-Anderson, thanks. 

Is the change you want something like this, shorter, not mentioning  `Microsoft.AspNetCore.App` and moving up front that server-side assemblies are provided so no action to take?

(Example 1, short, you have the server-side assemblies already installed):
ASP.NET Core SignalR server-side assemblies are now installed with the .NET Core SDK and runtime installers. See [Remove obsolete package references](#remove-obsolete-package-references) in this document.

(Example2, clarifying for your version, you have the server-side assemblies already, then the detail on where it is now. ):
In ASP.NET Core 3.0 or later, SignalR server-side assemblies are installed with the .NET Core SDK and runtime installers. The server assemblies are included in the `Microsoft.AspNetCore.App` shared framework See [Remove obsolete package references](#remove-obsolete-package-references) in this document.

Sorry to have to ask for clarification, but those are my best guesses and what you want.   I looked through the discussion here several times and I was not clear on where you "mention what I said previously about `Microsoft.AspNetCore.App`" and what this text should change to exactly.  I do understand the why.